cmd/api/src/test/integration/harnesses.go

```diff
@@ -1399,6 +1399,79 @@ func (s *ADCSGoldenCertHarness) Setup(graphTestContext *GraphTestContext) {
 
 }
 
+type WeakCertBindingAndUPNCertMappingHarness struct {
+	EnterpriseCA1 *graph.Node
+	EnterpriseCA2 *graph.Node
+	Computer1     *graph.Node
+	Computer2     *graph.Node
+	Computer3     *graph.Node
+	Computer4     *graph.Node
+	Computer5     *graph.Node
+	Domain1       *graph.Node
+	Domain2       *graph.Node
+	Domain3       *graph.Node
+}
+
+func (s *WeakCertBindingAndUPNCertMappingHarness) Setup(graphTestContext *GraphTestContext) {
+	domainSid1 := "S-1-5-21-2697957641-2271029196-387917394"
+	domainSid2 := "S-1-5-21-2697957641-2271029196-387917395"
+	domainSid3 := "S-1-5-21-2697957641-2271029196-387917396"
+
+	// Set up ECA nodes
+	s.EnterpriseCA1 = graphTestContext.NewActiveDirectoryEnterpriseCAWithThumbprint("EnterpriseCA1", domainSid1, "a")
+	s.EnterpriseCA2 = graphTestContext.NewActiveDirectoryEnterpriseCAWithThumbprint("EnterpriseCA2", domainSid2, "b")
```

Comment:
According to the attached svg/json representation of this harness, EnterpriseCA2 should be associated with domainSid3 instead of domainSid2.